bundler/lib/bundler/man/gemfile.5.ronn

```diff
@@ -74,6 +74,11 @@ you can use the `file` option instead.
 
     ruby file: ".ruby-version"
 
+The version file should conform to any of the following formats:
+
+  - `3.1.2` (.ruby-version)
```

Comment:
It would make sense to support the same formats that `.ruby-version` supports, which includes `ruby-3.1.2` (note the dash).